cli/src/main/java/org/openjdk/skara/cli/GitJCheck.java

```diff
@@ -172,4 +178,8 @@ public static void main(String[] args) throws Exception {
             }
         }
     }
+
+    private static boolean isURL(String pathOrURL) {
+        return urlPattern.matcher(pathOrURL).matches();
```

Comment:
The regex solution works fine, I just tend to stick with `String::startsWith` for simple cases :smiley: 
What do you think of `return s.startsWith("http://") || s.startsWith("https://");` ? I'm fine with the regex if you prefer that.

Reply:
I was using `s.matches("^http://")` before, but checking on the URI specification https://tools.ietf.org/html/rfc3986#section-3.1

> Although schemes are case-
   insensitive, the canonical form is lowercase

That's why I went with the case insensitive regex. If we don't care about case then it's fine to use `startsWith`, but otherwise we'd need to check every combination of upper and lower case, so in that case the regex seems better.

Reply:
Ah, yes, you are right, better stick with the regex :